docs/install.rst

```diff
@@ -434,4 +434,4 @@ installing astropy) is to run this from the root of the source tree::
 
 There are also alternative methods of :ref:`running-tests`.
 
-.. include:: development/workflow/known_projects.inc
\ No newline at end of file
+.. include:: development/workflow/known_projects.inc
```

Comment:
I don't understand why there is a diff here. Tab/space confusion? Trailing space?